.changes/c2af8534-4505-415d-bafa-ab035e261c08.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "c2af8534-4505-415d-bafa-ab035e261c08",
+    "type": "misc",
+    "description": "Upgrade dependencies to their latest versions, notably Kotlin 1.9.20"
+}
```

Comment:
**Question**: Do we call changes like this "breaking"? Should we minor-version-bump the CRT's `sdkVersion`?